tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -0,0 +1,40 @@
+""" Unit tests for Riemannian Mean Shift method """
```

Comment:
🎉 

Missing "." at the end of the module's docstring. 

@SaitejaUtpala do you know why DeepSource is not picking up on these docstrings' conventions?